docker/README.md

```diff
@@ -58,7 +80,22 @@ ACRA_DOCKER_IMAGE_TAG="2d2348f440aa0c20b20cd23c49dd34eb0d42d6a5"
 ACRA_DOCKER_IMAGE_TAG="0.76-33-g8b16bc2"
 ```
 
+Please define database name and user credentials:
+```
+# for Postgresql
+export POSTGRES_DB="<db_name>"
+export POSTGRES_USER="<user_name>"
+export POSTGRES_PASSWORD="<user_password>"
+
+# for MySQL
+export MYSQL_ONETIME_PASSWORD="<mysql_onetime_password>"
+export MYSQL_ROOT_PASSWORD="<mysql_root_password>"
+export MYSQL_DATABASE="<db_name>"
+export MYSQL_USER="<user_name>"
+export MYSQL_PASSWORD="<user_password>"
```

Comment:
how about `MYSQL_USER_PASSWORD`?

Reply:
The variable `MYSQL_PASSWORD` is inherited from original mysql container configuration. So I think it would be better to keep naming untouched.